application/helpers/admin/statistics_helper.php

```diff
@@ -380,7 +382,7 @@ function buildSelects($allfields, $surveyid, $language)
                 $thisquestion = Yii::app()->db->quoteColumnName($pv) . " IN (";
 
                 foreach ($_POST[$pv] as $condition) {
-                    $thisquestion .= "'$condition', ";
+                    $thisquestion .= "'" . getEncryptedCondition($responseModel, $pv, $condition) . "', ";
```

Comment:
usage of https://www.yiiframework.com/doc/api/1.1/CDbConnection#quoteValue-detail seems better, no ?